themes/gatsby-theme-blog-core/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "gatsby-theme-blog-core",
+  "version": "1.0.0",
+  "main": "index.js",
+  "author": "christopherbiscardi <chris@christopherbiscardi.com> (@chrisbiscardi)",
+  "license": "MIT",
+  "peerDependencies": {
+    "gatsby": "^2.13.13",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
+  },
+  "dependencies": {
+    "gatsby-plugin-mdx": "^1.0.16",
+    "gatsby-plugin-sharp": "^2.2.9",
+    "gatsby-remark-copy-linked-files": "^2.1.3",
+    "gatsby-remark-images": "^3.1.7",
+    "gatsby-remark-numbered-footnotes": "^1.0.0",
+    "gatsby-remark-smartypants": "^2.1.2",
+    "gatsby-source-filesystem": "^2.1.6",
+    "gatsby-transformer-sharp": "^2.2.4",
+    "remark-slug": "^5.1.2"
+  },
+  "devDependencies": {
+    "@mdx-js/mdx": "^1.1.0",
```

Comment:
Just took this for a spin and it errored out here due to the lack of `@mdx-js/mdx`. I _think_ we can include this dependency because it won't be loaded on the client or cause duplicate contexts to stomp eachother like `@mdx-js/react` would.

(after manually adding this dep it's all working out great tho!)

Reply:
yeah, I guess you could always upgrade by forcing resolutions if you needed a bugfix. Same for upgrading child themes and such.